### DISCUSSION TITLE
Can someone help me understand why the function "wordf" gives a "Ignoring exception in command None: discord.ext.commands.errors.CommandNotFound:"

### DISCUSSION THREAD

**Discussion by DevonDekhran** [Rapptz/discord.py#5988] [Q&A]
import discord
from discord.ext import commands
from discord.ext.commands import Bot
import json
import difflib
from difflib import get_close_matches

@bot.command()
async def wordf(ctx, word):
await ctx.send(getTheWord(word))

bot.run('')

**Comment by DevonDekhran** [comment]
Someone?

**Comment by iiPythonx** [comment]
I'm assuming you are running the command `word` inside of discord but your function is actually called `wordf`.
You could add a `name = 'word'` inside of your `@bot.command()` and see if that fixes the problem.

**Comment by scrazzz** [comment]
`from discord.ext.commands import Bot` is redundant by the way.
